mne/io/fiff/tests/test_raw_fiff.py

```diff
@@ -1141,7 +1142,7 @@ def test_add_channels():
     assert_raises(ValueError, raw_meg.copy().add_channels, [raw_arr])
     raw_meg.copy().add_channels([raw_arr], force_update_info=True)
     # Make sure that values didn't get overwritten
-    assert_true(raw_arr.info['dev_head_t'] is orig_head_t)
+    _is_equal_dict([raw_arr.info['dev_head_t'], orig_head_t])
```

Comment:
don't you need to `assert_true(_is_equal_dict(...))`? If not, then that is a misnomer and should be renamed `_assert_equal_dict`